website/config/custom.js

```diff
@@ -96,33 +96,34 @@ module.exports.custom = {
   ***************************************************************************/
   githubRepoDRIByPath: {
     'README.md': 'chris-mcgillicuddy',// (github brandfront)
+    'tools/fleetctl-npm/README.md': ['chris-mcgillicuddy', 'mike-j-thomas'],//« brandfront for fleetctl package on npm
 
-    'handbook': ['desmi-dizney', 'mike-j-thomas', 'mikermcneil'],// (default for handbook)
+    'articles': ['chris-mcgillicuddy', 'mike-j-thomas', 'eashaw'],
+    
+    'handbook': ['chris-mcgillicuddy', 'mike-j-thomas', 'eashaw', 'mikermcneil'],// (default for handbook)
     'handbook/company': 'mikermcneil',
     'handbook/people': 'charlottechance',
     'handbook/engineering': 'zwass',
     'handbook/product': 'noahtalerman',
     'handbook/security': 'guillaumeross',
+    'handbook/sales': 'alexmitchelliii',
     'handbook/digital-experience': 'mike-j-thomas',
     'handbook/growth': 'timmy-k',
     'handbook/customers': 'tgauda',
-    'handbook/community': ['dominuskelvin', 'ksatter'],
-    'handbook/README.md': '*',// (any fleetie can update this page and merge their change without waiting for their change to be approved)
-
-    'tools/fleetctl-npm/README.md': ['mike-j-thomas'],//« brandfront for fleetctl package on npm
+    'handbook/community': ['timmy-k'],
 
     'website': 'mikermcneil',// (default for website)
     'website/views': 'eashaw',
     'website/assets': 'eashaw',
     'website/config/routes.js': ['eashaw', 'mike-j-thomas'],// (for managing website URLs)
 
-    'docs': 'zwass',// (default for docs)
-    'docs/images': ['noahtalerman', 'eashaw', 'mike-j-thomas'],
-    'docs/Using-Fleet/REST-API.md': 'lukeheath',
-    'docs/Contributing/API-for-contributors.md': 'lukeheath',
-    'docs/Deploying/FAQ.md': ['ksatter', 'dominuskelvin'],
-    'docs/Contributing/FAQ.md': ['ksatter', 'dominuskelvin'],
-    'docs/Using-Fleet/FAQ.md': ['ksatter', 'dominuskelvin'],
+    'docs': ['desmi-dizney', 'chris-mcgillicuddy'],// (default for docs)
```

Comment:
Can we leave me on docs? I often want to make quick doc changes when I notice issues.

Reply:
Def

Reply:
(did the same for articles and the standard query library)